Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'activerecord', '~> 7.0.4' # Rails. ORM and query system.
 gem 'activesupport', '~> 7.0.4' # Rails. Underlying library.
 # gem 'activetext' # Not used. Text editor that fails to support markdown.
 gem 'attr_encrypted', github: 'andrewfader/attr_encrypted'
-gem 'bcrypt', '~> 3.1.18' # Security - for salted hashed interated passwords
+gem 'bcrypt', '~> 3.1.18' # Security - for salted hashed interacted passwords
```

Comment:
here is a merge conflict I guess now, but I also recalled that for this fix I pretty much chose only with a hint of sense (interacted entry) but I could have been wrong and initial intention was something else?

Reply:
BTW -- do you prefer me to merge `main` (might be preferable to minimize re-review) or to rebase to resolve the conflict?

Reply:
I don't have a *strong* opinion, but rebasing seems unnecessary. We can fix a 1-line merge conflict. I primarily want to ensure we don't lose your fine work :-).